m2cgen/interpreters/interpreter.py

```diff
@@ -15,26 +15,24 @@ def interpret(self, expr):
     # Default method implementations
 
     def interpret_if_expr(self, expr, if_var_name=None,
-                          is_multi_output=False, **kwargs):
+                          _is_vector_output=False, **kwargs):
```

Comment:
Considering that we can retrieve `is_vector_output` value from any expression, do we still need this argument?